roomee-api/roomee-api/Models/Event.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Security.Claims;
+
+namespace roomee_api.Models
+{
+    public class Event{
+        [JsonProperty(PropertyName = "eventId")]
+        public int EventId { get; }
+
+        [JsonProperty(PropertyName = "creatorUserId")]
+        public int CreatorUserId { get; set; }
+
+        [JsonProperty(PropertyName = "roomId")]
+        public int RoomId { get; set; }
+
+        [JsonProperty(PropertyName = "creationTimestamp")]
+        public DateTime CreationTimestamp { get; set; }
+
+        [JsonProperty(PropertyName = "startTimestamp")]
+        public DateTime StartTimestamp { get; set; }
+
+        [JsonProperty(PropertyName = "endTimestamp")]
+        public DateTime EndTimestamp { get; set; }
+
+        [JsonProperty(PropertyName = "title")]
+        public string Title { get; set; }
+
+        [JsonProperty(PropertyName = "description")]
+        public string Description { get; set; }
+
+        [JsonProperty(PropertyName = "statusId")]
+        public int StatusId { get; }
+
+        public static readonly string[] UpdateNames = { "creatorUserId", "roomId", "creationTimestamp", "startTimestamp", "endTimestamp", "title", "description", "statusId" };
```

Comment:
CreatorUserId and CreationTimestamp should not be able to be modified once the object is initially created by the user.